### PR TITLE
multi: show more bond info on dexsettings page

### PR DESCRIPTION
**Pull request by ukane-philemon**
UI changes:
Screenshot 2023-08-23 at 3 16 09 AM

### CODE REVIEW

**Review by JoeGruffins — APPROVED** [review]
Could add some more scroll over info? For example:
Current Tier "Number of active bonds that have not yet reached the expiry threshold."
Pending Bonds "Number of bonds that will be active when enough on chain confirmations have been reached. These do not yet count towards the current tier."
Expired Bonds "Bonds that are still locked but have passed the expiry threshold. These do not count towards current tier and will be refunded."

**Comment by ukane-philemon** [comment]
> Could add some more scroll over info? 

Sure! Thanks for reviewing.

**Comment by ukane-philemon** [comment]
With tooltips:

Screenshot 2023-08-28 at 9 18 20 PM

File: client/webserver/site/src/html/dexsettings.tmpl [hunk 1]
```diff
@@ -46,24 +46,44 @@
...
+          <span class="ico-info" data-tooltip="[[[pending_bonds_tooltip]]]"></span>
+          <br>
+          <span id="pendingBonds">10</span>
+        </span>
+        <span class="text-center">
+          [[[expired_bonds]]]
+          <span class="ico-info" data-tooltip="[[[expired_bonds_tooltip]]]"></span>
+          <br>
+          <span id="expiredBonds">10</span>
+        </span>
```
Comment (buck54321) [inline client/webserver/site/src/html/dexsettings.tmpl:71]: I don't think a normal user would find this information useful. It risks confusing people. How about we start with a table of some sort, showing, for all bond assets, 
1. Cost of a single bond, with fiat conversion if available
2. Total locked amount for single bond, with fiat conversion if available
3. Current target tier and server reported tier

We could also provide some basic info about how trading limits will scale with bond tier, and that additional bonds can offset penalties.

Comment (ukane-philemon) [reply client/webserver/site/src/html/dexsettings.tmpl:71]: Sure, we can provide additional information on this page. I recommend displaying information for one asset at a time. Users can easily choose another asset, and we will then display its corresponding information.

Comment (buck54321) [reply client/webserver/site/src/html/dexsettings.tmpl:71]: After I bond for 1 tier, I want to know how much is going to be unavailable to me once I bond. Something like `bondAmt * targetTier * 2 + bondsFeeBuffer`.

Comment (ukane-philemon) [reply client/webserver/site/src/html/dexsettings.tmpl:71]: > After I bond for 1 tier, I want to know how much is going to be unavailable. Something like `bondAmt * targetTier * 2 + bondsFeeBuffer`.

Oh, this. Okay.

**Comment by ukane-philemon** [comment]
@buck54321, I've addressed the requirements as mentioned here(https://github.com/decred/dcrdex/pull/2485#discussion_r1312413384). Please let me know if I missed anything.

https://github.com/decred/dcrdex/assets/57448127/a2a0f8cb-8065-457e-8e28-d1f5b9636519

PS: Saw some wired webview log here: https://github.com/ukane-philemon/dcrdex/actions/runs/6248690677/job/16963818304

File: client/webserver/api.go [hunk 2]
```diff
@@ -424,24 +424,30 @@ func (s *WebServer) apiPostBond(w http.ResponseWriter, r *http.Request) {
...
+
+	if post.FeeBuffer != nil {
+		bondForm.FeeBuffer = *post.FeeBuffer
+	} else {
+		feeBuffer, err := s.bondsFeeBuffer(assetID) // or let wallet use internal feeBuffer?
+		if err != nil {
+			s.writeAPIError(w, err)
+			return
+		}
+		post.FeeBuffer = &feeBuffer
```
Comment (buck54321) [inline client/webserver/api.go:447]: I don't understand this line. Is it supposed to be `bondForm.FeeBuffer = feeBuffer`? Why are we doing this in **webserver** and not **core**?

Comment (ukane-philemon) [reply client/webserver/api.go:447]: > Is it supposed to be bondForm.FeeBuffer = feeBuffer

Oh, yes.

> Why are we doing this in webserver and not core?

We've always fetched the feeBuffer here and I have no objection to moving it to `core` level.

File: client/webserver/site/src/js/doc.ts [hunk 3]
```diff
@@ -448,6 +449,15 @@ export default class Doc {
...
   }
+
+  // showFiatValue displays the fiat equivalent for the provided amount.
+  static showFiatValue (assetID: number, amount: number, display: PageElement): void {
+    const rate = app().fiatRatesMap[assetID]
+    if (rate) {
+      display.textContent = Doc.formatFiatConversion(amount, rate, app().unitInfo(assetID))
+      Doc.show(display.parentElement as Element)
+    } else Doc.hide(display.parentElement as Element)
+  }
```
Comment (buck54321) [inline client/webserver/site/src/js/doc.ts:460]: **doc.ts** shouldn't import `app`. This is off-pattern for these types of methods anyway. Try https://github.com/buck54321/dcrdex/compare/90ab9f5...buck54321:dcrdex:showfiatvalue

**Review by buck54321 — CHANGES_REQUESTED** [review]

File: client/webserver/site/src/js/doc.ts (hunk 3 above)
Comment (ukane-philemon) [reply client/webserver/site/src/js/doc.ts]: Thanks. I cherry-picked your commit.

**Pull request merged**